ReflectionTest/ReflectionConsoleTest/Program.cs

```diff
@@ -12,62 +12,61 @@ class Program
     {
         static void Main(string[] args)
         {
-            try
+            var user = new User("tsar", "12345", true);
+            var users = new UserStorage();
+
+            for (int i = 0; i < 20; i++)
             {
-                var user = new User("tsar", "12345", true);
-                var users = new UserStorage();
+                users.AddRandomUser();
+            }
 
-                for (int i = 0; i < 20; i++)
-                {
-                    users.AddRandomUser();
-                }
+            users.AddUser(user);
+            users.DeleteUser(users.FindByLogin("tsar"));
+            users.DeleteUser(user);
 
-                users.AddUser(user);
-                users.DeleteUser(users.FindByLogin("tsar"));
-                users.DeleteUser(user);
-                users.DeleteUser(user);
+            GetReflectionInfo("SomeLibrary");
 
-                
-                var assemblies = AppDomain.CurrentDomain.GetAssemblies();
-                Type[] typelist = new Type[0];
-                foreach (var assembly in assemblies)
-                {
-                    typelist = assembly.GetTypes().Where(t => t.Namespace == "SomeLibrary").ToArray();
+            Console.WriteLine("Press any key...");
+            Console.ReadKey();
+        }
 
-                    if (typelist.Any())
-                    {
-                        break;
-                    }
-                }
+        public static bool GetReflectionInfo(string assemblyNamespace)
+        {
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+            IEnumerable<Type> types = new Type[0];
+            foreach (var assembly in assemblies)
+            {
+                types = assembly.GetTypes().Where(t => t.Namespace == assemblyNamespace);
 
-                if (!typelist.Any())
+                if (types.Any())
                 {
-                    throw new Exception("Сan't find the target assembly.");
+                    break;
                 }
+            }
+
+            if (!types.Any())
+            {
+                return false;
+            }
 
```

Comment:
No need to break or return false. If Where returns empty enumerable you'll just never get into foreach.

Reply:
Okay, I'll remember that. But break was used to ensure that the value types were not assigned null at further passes.